assets/js/components/App.tsx

```diff
@@ -23,11 +25,10 @@ class AppRoutes extends React.Component {
             path="/emergency-takeover"
             element={<OutfrontTakeoverTool />}
           ></Route>
-          <Route
-            path="/dashboard"
-            element={<Dashboard page="places" />}
-          ></Route>
-          <Route path="/alerts" element={<Dashboard page="alerts" />}></Route>
+          <Route path="*" element={<Dashboard />}>
+            <Route path="dashboard" element={<PlacesPage />}></Route>
+            <Route path="alerts" element={<AlertsPage />}></Route>
```

Comment:
I've been working on that today in a different branch and it's not easy to find a good place for the `Outlet` to live on the `AlertsPage`. I'm going to try to make it work, but it may be easier to make it its own `Route` to reduce the amount of conditional elements that would need to live in the `AlertsPage`.